uqcsbot/scripts/xkcd.py

```diff
@@ -0,0 +1,75 @@
+import requests
+import feedparser
+from urllib.parse import quote
+from uqcsbot import bot, Command
+from uqcsbot.utils.command_utils import loading_status
+
+
+# HTTP Endpoints
+XKCD_BASE_URL = "https://xkcd.com/"
+XKCD_RSS_URL = "https://xkcd.com/rss.xml"
+RELEVANT_XKCD_URL = 'https://relevantxkcd.appspot.com/process?action=xkcd&query='
+
+
+def get_by_id(comic_number: int) -> str:
+    """
+    Gets an xkcd comic based on its unique ID/sequence number.
+    :param comic_number: the ID number of the xkcd comic to retrieve.
+    :return: a response containing either a comic URL or an error message.
+    """
+    if comic_number < 0:
+        return "Invalid xkcd ID, it must be a positive integer."
```

Comment:
`< 0` implies non-negative, should it be `<=`?

Reply:
so that sounds like the fail case should be `<= 0` then? :P my comment was more-so to do with the statement "it must be a positive integer", of which 0 is not ofc. Unless i'm missing something I think this should be `<= 0`

Reply:
Right, my bad. I completely misunderstood what you were getting at. Fixed.